src/lib.rs

```diff
@@ -2308,9 +2308,12 @@ impl Build {
                             None
                         };
 
-                    let clang_target =
-                        target.llvm_target(&self.get_raw_target()?, version.as_deref());
+                    let rustc_target = self.get_raw_target()?;
+                    let clang_target = target.llvm_target(&rustc_target, version.as_deref());
                     cmd.push_cc_arg(format!("--target={clang_target}").into());
+                    if rustc_target == "wasm32v1-none" {
+                        cmd.push_cc_arg("-mcpu=mvp".into());
+                    }
```

Comment:
It'd be great if we can avoid having duplicate code.

Maybe we can have another match and if below this block?

Reply:
I'm afraid it will be difficult to debug if someone needs it